lib/Listener/AlbumsManagementEventListener.php

```diff
@@ -3,18 +3,21 @@
 namespace OCA\Photos\Listener;
 
 use OCA\Photos\Album\AlbumMapper;
-use OCP\Files\File;
-use OCP\Files\Folder;
-use OCP\Files\Node;
 use OCP\EventDispatcher\Event;
 use OCP\EventDispatcher\IEventListener;
 use OCP\Files\Events\Node\NodeDeletedEvent;
+use OCP\Files\File;
+use OCP\Files\Folder;
+use OCP\Files\Node;
 use OCP\Group\Events\GroupDeletedEvent;
 use OCP\Group\Events\UserRemovedEvent;
 use OCP\Share\Events\ShareDeletedEvent;
 use OCP\User\Events\UserDeletedEvent;
 use Psr\Log\LoggerInterface;
 
+/**
+ * @template-implements IEventListener<Event|NodeDeletedEvent|GroupDeletedEvent|ShareDeletedEvent|UserDeletedEvent>
```

Comment:
```suggestion
 * @template-implements IEventListener<NodeDeletedEvent|GroupDeletedEvent|ShareDeletedEvent|UserDeletedEvent>
```
is `Event` really needed in there?

Reply:
No idea, it works with it, and does not hurt